flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/BoundedBlockingSubpartitionWriteReadTest.java

```diff
@@ -151,16 +162,24 @@ public void testRead10ConsumersConcurrent() throws Exception {
 	//  common test passes
 	// ------------------------------------------------------------------------
 
-	private static void readLongs(ResultSubpartitionView reader, long numLongs, int numBuffers) throws Exception {
+	private static void readLongs(
+			ResultSubpartitionView reader,
+			long numLongs,
+			int numBuffers,
+			boolean compressedEnabled,
+			BufferDecompressor decompressor) throws Exception {
```

Comment:
Do not need to change the arguments because they can be referenced directly anywhere inside class

Reply:
for the multi-thread test, each thread need a decompressor, because BufferDecompressor is not thread safe.